src/admin-views/tribe-options-display.php

```diff
@@ -3,28 +3,153 @@
+			'description of settings section',
+			'event-tickets'
+		),
+		tribe_get_rsvp_label_plural( 'tickets_rsvp_display_description' )
+	)
+);
+
+if ( $etp_active ) {
+	$tickets_rsvp_display_title = esc_html(
+		sprintf(
+		// Translators: %1$s: dynamic "Tickets" text, %2$s: dynamic "RSVPs" text.
+			_x(
+				'%1$s and %2$s Display Settings',
+				'title of settings section',
+				'event-tickets'
+			),
+			esc_html( tribe_get_ticket_label_plural( 'tickets_rsvp_display_title' ) ),
+			tribe_get_rsvp_label_plural( 'tickets_rsvp_display_title' )
+		)
+	);
+
+	$tickets_rsvp_display_description = esc_html(
+		sprintf(
+		// Translators: %1$s: dynamic "Tickets" text, %2$s: dynamic "RSVPs" text.
+			_x(
+				'The settings below control the display of your %1$s and %2$s.',
+				'description of settings section',
+				'event-tickets'
+			),
+			esc_html( tribe_get_ticket_label_plural( 'tickets_rsvp_display_description' ) ),
+			tribe_get_rsvp_label_plural( 'tickets_rsvp_display_description' )
+		)
+	);
+}
+
+$et_options_display = [
+	'tickets_rsvp_display_title'       => [
 		'type' => 'html',
-		'html' => '<h3>' . __( 'RSVP Display Settings', 'event-tickets' ) . '</h3>',
+		'html' => '<h3 id="tickets_rsvp_display_title">' . $tickets_rsvp_display_title . '</h3>',
 	],
-	'rsvp-display-description'   => [
+	'tickets_rsvp_display_description' => [
 		'type' => 'html',
-		'html' => '<p>' . __( 'The settings below control the display of your RSVPs.', 'event-tickets' ) . '</p>',
+		'html' => '<p>' . $tickets_rsvp_display_description . '</p>',
 	],
-	'tickets_rsvp_use_new_views' => [
+];
+
+// Only show this option to older installs, as newer installs default to new views.
+if ( tribe_installed_before( 'Tribe__Tickets__Main', '5.0' ) ) {
+	$et_options_display['tickets_rsvp_use_new_views'] = [
 		'type'            => 'checkbox_bool',
-		'label'           => __( 'Enable New RSVP Experience', 'event-tickets' ),
-		'tooltip'         => __( 'This setting will render the new front-end designs (styling) and user-flow for the RSVP experience.', 'event-tickets' ),
+		'label'           => esc_html(
+			sprintf(
+			// Translators: %1$s: dynamic "RSVPs" text.
+				_x(
+					'Enable New %1$s Experience',
+					'settings label',
+					'event-tickets'
+				),
+				tribe_get_rsvp_label_singular( 'tickets_rsvp_use_new_views' )
+			)
+		),
+		'tooltip'         => esc_html(
+			sprintf(
+			// Translators: %1$s: dynamic "RSVPs" text.
+				_x(
+					'This setting will render the new front-end designs (styling) and user-flow for the %1$s experience.',
+					'settings tooltip',
+					'event-tickets'
+				),
+				tribe_get_rsvp_label_singular( 'tickets_rsvp_use_new_views' )
+			)
+		),
 		'validation_type' => 'boolean',
 		'default'         => false,
-	],
-] );
+	];
+}
+
+// Only show this option to older installs, as newer installs default to new views.
+if (
+	$etp_active
+	&& tribe_installed_before( 'Tribe__Tickets_Plus__Main', '5.1' )
```

Comment:
This is fine for now, but at some point we'll want ET to have access to enabling the new views too. Nothing to change here, just a note for our future selves.

Reply:
Curious the long-term thought here

Reply:
These are using the new v2 views in ET itself, which have their own benefits, including support for new ET features/tweaks/filters we may add in the future too.